setup.py

```diff
@@ -114,6 +104,11 @@ def readfile(fname):
     import wheelhouse_uploader.cmd
     cmdclass.update(vars(wheelhouse_uploader.cmd))
 
+python_2_6_backports = ''
```

Comment:
The empty string doesn't cause any output in Travis.
https://travis-ci.org/piskvorky/gensim/jobs/118421963#L290